valve_control:
  ros__parameters:
    sim: true

vacuum_sensor:
  ros__parameters:
    sim: true
